db/migrate/20220914105820_create_posts.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :title
       t.string :text
       t.integer :comments_counter
-      t.integer :likes_counter      
+      t.integer :likes_counter
       t.references :author, index: true, foreign_key: { to_table: :users }
 
       t.timestamps
```

Comment:
- [ ] Kindly make sure you fix the following error like in the image below as right now, I cannot run the migrations for your application and even test the application to see all the wireframes design as required for the project.
![Screenshot from 2022-09-23 18-24-07](https://user-images.githubusercontent.com/83042742/192008281-d7094091-3fdf-44a9-b449-e2e3772013dd.png)